molecule/tests/roles/confd/variables/haproxy.tmpl.yml

```diff
@@ -0,0 +1,162 @@
+---
+# 🚀 These tasks aim to test the "ansible.builtin.lineinfile" task
+# 🎯 The objective is to ensure that the lines are correctly replaced
+
+# 📂 Ensure tmp directory exists
+- name: Molecule.tests.roles.confd.variables.haproxy.tmpl | Ensure tmp directory exists
+  run_once: true
+  delegate_to: localhost
+  ansible.builtin.file:
+    path: "./tmp"
+    state: directory
+
+# 🔄 Define a dummy template file
+- name: Molecule.tests.roles.confd.variables.haproxy.tmpl | Set template file test data
+  ansible.builtin.set_fact:
+    haproxy_listen_port:
+      stats: 8080
+      master: 8081
+      replicas: 8082
+      replicas_sync: 8083
+      replicas_async: 8084
```

Comment:
maybe we will use the same ports here as in the main code (500x ports)?

Reply:
Indeed, that is a good idea

Reply:
Done